plugins/catalog-backend-module-backstage-openapi/src/InternalOpenApiDocumentationProvider.ts

```diff
@@ -229,13 +229,22 @@ export class InternalOpenApiDocumentationProvider implements EntityProvider {
     const pluginsToMerge = this.config.getStringArray(
       'catalog.providers.backstageOpenapi.plugins',
     );
-    logger.info(`Loading specs from from ${pluginsToMerge}.`);
+
+    const getOverride = (key: string, fallbackValue: string = '') => {
+      return (
+        this.config.getOptionalString(
+          `catalog.providers.backstageOpenapi.entityOverrides.${key}`,
+        ) ?? fallbackValue
+      );
+    };
```

Comment:
This utility function tries to load the value from the config file for the key provided in the first argument, or falls back to a default string value provided as the second argument.